evalml/tests/data_checks_tests/test_invalid_targets_data_check.py

```diff
@@ -193,3 +196,78 @@ def test_invalid_target_data_check_n_unique():
                                   message_code=DataCheckMessageCode.TARGET_BINARY_NOT_TWO_UNIQUE_VALUES,
                                   details={"target_values": unique_values}).to_dict()]
     }
+
+
+@pytest.mark.parametrize("objective", ['Root Mean Squared Log Error', 'Mean Squared Log Error', 'Mean Absolute Percentage Error'])
```

Comment:
Can you add some test cases where we pass an instance of the objective as opposed to the name? That's technically allowed so we should cover those cases!